lib/serial_terminal.pm

```diff
@@ -69,26 +69,37 @@ for ppc64le).
 =cut
 
 sub prepare_serial_console {
-    record_info('getty before', script_output('systemctl | grep serial-getty'));
+    # Virtio console is disabled only with VIRTIO_CONSOLE=0 (by default
+    # VIRTIO_CONSOLE is not set which also means that virtio console is
+    # *enabled*).
+    unless (!check_var('VIRTIO_CONSOLE', 0)) {
+        record_info('skip virtio', 'Skipped due disabled virtio console');
+        return;
+    }
 
-    if (!check_var('VIRTIO_CONSOLE', 0)) {
-        my $console = 'hvc1';
+    unless (is_qemu) {
+        record_info('skip backend', 'Skipped due unsupported arch (only BACKEND=qemu supported)');
```

Comment:
```suggestion
        record_info('skip backend', 'Skipped due unsupported backend (only BACKEND=qemu supported)');
```

Maybe it makes also sense to state *what* has been skipped (not sure whether it'll be clear in the context the info is recorded):
```suggestion
        record_info('skip virtio', 'Skipped adding consoles due unsupported backend (only BACKEND=qemu supported)');
```

Reply:
Thank you, both makes sense. I had different title `skip virtio` vs. `skip backend` to see the reason without click. But I'll change it (it's better to click that have misleading title).